packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTabs.tsx

```diff
@@ -33,6 +34,31 @@ export const StocktakeLineEditTabs: FC<{
   const t = useTranslation('inventory');
   const [currentTab, setCurrentTab] = useState(Tabs.Batch);
 
+  useEffect(() => {
+    const keybindings = (e: KeyboardEvent) => {
+      if (e.code === 'Digit1' && e.shiftKey) {
+        e.preventDefault();
+        setCurrentTab(Tabs.Batch);
+      }
+      if (e.code === 'Digit2' && e.shiftKey) {
+        e.preventDefault();
+        setCurrentTab(Tabs.Pricing);
+      }
+      if (e.code === 'Digit3' && e.shiftKey) {
+        e.preventDefault();
+        setCurrentTab(Tabs.Location);
+      }
+      if (e.code === 'Equal' && e.shiftKey) {
+        e.preventDefault();
+        onAddLine();
+      }
+    };
+
+    window.addEventListener('keydown', keybindings);
+
+    return () => window.removeEventListener('keydown', keybindings);
+  }, []);
```

Comment:
Just took this from inbound. There's likely far better methods of adding shortcuts for react apps. possibly even just utilising kbar to do it